src/js/samples/components/Sidebar/Selector.js

```diff
@@ -69,10 +74,12 @@ export const SampleSidebarSelector = ({
                             autoFocus
                         />
                     </SampleSidebarSelectorInputContainer>
-                    {sampleItemComponents}
-                    <StyledLinkButton>
-                        <Link to={manageLink}> Manage</Link>
-                    </StyledLinkButton>
+                    <StyledDiv>
```

Comment:
`StyledLinkButton` is incorrectly named, so don't follow that as an example.

The Styled Components convention is to use the `Styled` prefix only in the following situation:
```js
const StyledBar = styled.div`...`;

const Bar = props => (
  <StyledBar>
    ...
  </StyledBar>
);
```

https://styled-components.com/docs/basics#how-do-styled-components-work-within-a-component

---

Can you rename `StyledLinkButton` to `SampleSidebarSelectorButton`?